dynamo/vectorfield/scVectorField.py

```diff
@@ -440,20 +445,16 @@ def SparseVFC(
         beta = 1 / h ** 2
 
     K = (
-        con_K(ctrl_pts, ctrl_pts, beta, timeit=timeit_)
-        if div_cur_free_kernels is False
-        else con_K_div_cur_free(ctrl_pts, ctrl_pts, sigma, eta, timeit=timeit_)[0]
-    )
-    U = (
-        con_K(X, ctrl_pts, beta, timeit=timeit_)
+        con_K(ctrl_pts, ctrl_pts, beta)
```

Comment:
the nice thing for timeit_ is that it will show the time only when timeit_ is used. Can we also make something like  this is the case for your logging time?